packages/datadog-plugin-mongodb-core/test/integration-test/client.spec.js

```diff
@@ -19,26 +19,26 @@ describe('esm', () => {
   let proc
   let sandbox
 
-  before(async function () {
-    this.timeout(20000)
-    sandbox = await createSandbox(['mongodb', 'mongodb-core'], false, [
-      `./packages/datadog-plugin-mongodb-core/test/integration-test/*`])
-  })
+  withVersions('mongodb-core', 'mongodb', '>=4', version => {
```

Comment:
Why the explicit range here?

Reply:
Also looks like mongodb and mongodb-core got reversed?

Reply:
Do you mean the server.mjs file?

Reply:
If so then the server.mjs file was updated to work better with newer mongodb versions